.editorconfig

```diff
@@ -41,7 +41,7 @@ dotnet_sort_system_directives_first = true
 dotnet_separate_import_directive_groups = false
 
 # IDE0005 - Remove unnecessary imports
-dotnet_diagnostic.IDE0005.severity = suggestion
+dotnet_diagnostic.IDE0005.severity = warning
```

Comment:
I'm fine with an error in CI, as long as builds in VS/cli do not fail. As long as that holds, I'm good.

Reply:
VS: warning
<img width="1371" alt="image" src="https://github.com/dotnet/razor/assets/754264/1ab4d773-02e3-4239-9a49-8f51a16eda68">

CI: error (see build results 🤦‍♂️)

CLI: nothing
<img width="210" alt="image" src="https://github.com/dotnet/razor/assets/754264/3f145315-a3b1-422c-afdd-736b6fbf47bb">

I will have to dig in to that last one, as I suspect that would be really annoying for some folks.

Reply:
Scratch that, CLI reports errors for some reason.